homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
```

Comment:
In the assigment, it was hinted to use the (sorted) array index for the `value` attribute of the `option` element, e.g.:

```js
.forEach((repository, index) => {
  createAndAppend('option', select, { text: repository.name, value: index });
});
```

By using this index it is now easy to get the repository object from the `repositories` array that corresponds to the selected repo in your event handler in `main()`:

```js
select.addEventListener('change', () => {
  const repository = repositories[select.value];
  renderRepoBasisInfo(repository, ul);
  renderContributors(repository, ulContributions);
});
```

The variable `repository` is now no longer just the name of the selected repository but the repository object itself, containing all the details you need to render the data.

And of course, the object for the first repository to display is `repositories[0]`.